doc/backing-store-crd.md

```diff
@@ -5,7 +5,7 @@ BackingStore CRD represents a storage target to be used as underlying storage fo
 These storage targets are used to store deduped+compressed+encrypted chunks of data (encryption keys are stored separatly).
 Backing-stores are referred to by name when defining [BucketClass](bucket-class-crd.md).
 
-Multiple types of backing-stores are supported: aws-s3, s3-compatible, google-cloud-storage, azure-blob, obc, pvc.
+Multiple types of backing-stores are supported: aws-s3, s3-compatible, google-cloud-storage, azure-blob.
```

Comment:
Minor suggestion:

Multiple types of backing-stores are currently supported: aws-s3, s3-compatible, google-cloud-storage, azure-blob. Support for backing-store type pv-pool is not yet implemented.